docs/concepts.md

```diff
@@ -18,10 +18,25 @@ Its interface should be familiar to users of MySQL, Postgres, Oracle, Hive, Pres
 
 The KSQL CLI acts as a client to the KSQL server (see next section).
 
+>An alternative to the KSQL CLI is to use the KSQL Experimental UI.
+
+### KSQL Experimental UI
```

Comment:
This should be `####`.

Reply:
Also, please move this after the KSQL server section.

IMHO the ordering should be:
- KSQL CLI
- KSQL Server
- KSQL Experimental UI